shared/cfg/guest-hw.cfg

```diff
@@ -171,8 +183,8 @@ variants:
         nettype = bridge
         # Driver (kernel module) that supports SR-IOV hardware.
         # As of today (30-11-2009), we have 2 drivers for this type of hardware:
-        # Intel® 82576 Gigabit Ethernet Controller - igb
-        # Neterion® X3100™ - vxge
+        # Intel? 82576 Gigabit Ethernet Controller - igb
+        # Neterion? X3100? - vxge
```

Comment:
Apparently your editor has problems with utf-8 characters. Well I agree utf-8 characters should not be used in config files, but please fix it correctly and in separate commit. I'd suggest an extra commit where you replace `®` with `(R)` and `™` with (TM).